setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
```

Comment:
Shouldn't it be `0.1.0` to follow semver?

Not committing to stable is probably for the best indeed :)